src/goto-programs/goto_convert.cpp

```diff
@@ -766,7 +766,10 @@ void goto_convertt::convert_assign(
   else if(rhs.id()==ID_side_effect &&
           rhs.get(ID_statement)==ID_java_new)
   {
-    copy(code, ASSIGN, dest);
+    Forall_operands(it, rhs)
```

Comment:
Perhaps we can use range based for loop here?

Reply:
Going to stick with a plain revert there